src/Common/MetricHelper.cs

```diff
@@ -481,17 +481,25 @@ public void FlushMetric()
                 return;
             }
 
-            try
+            FlushMetricAsync(TelemetryClients);
```

Comment:
A warning will be throwed when we don't `await` an `async` method, the recommended way to suppress the warning is by assigning the returned task to a variable.

> You should consider suppressing the warning only if you're sure that you don't want to wait for the asynchronous call to complete and that the called method won't raise any exceptions. In that case, you can suppress the warning by assigning the task result of the call to a variable.

https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/compiler-messages/cs4014

```suggestion
            Task flushInBackground = FlushMetricAsync(TelemetryClients);
```

Reply:
Updated.